tests/testthat/test-carehomes-pmcmc.R

```diff
@@ -214,7 +214,7 @@ test_that("can combine rt calculations over trajectories without reordering", {
 test_that("adding incidence adds appropriate states - nested", {
   dat <- reference_data_mcmc()
   dat$trajectories$state <- array(
-    dat$trajectories$state, c(129, 11, 2, 32),
+    dat$trajectories$state, c(111, 11, 2, 32),
```

Comment:
why has this changed? just curious?

Reply:
I believe because prob strain went from along groups (19) to not (1) so 18 difference